variable/include/scipp/variable/bucket_model.h

```diff
@@ -126,6 +126,16 @@ template <class T> class DataModel<bucket<T>> : public VariableConcept {
             return x.second >= 0 && x.first > x.second;
           }) != vals.end())
         throw except::SliceError("Overlapping bucket indices are not allowed.");
+      if (buffer.dims().ndim() > 1 && buffer.dims().index(dim) != 0) {
+        if (std::find_if(vals.begin(), vals.end(), [](const auto x) {
+              return x.first == x.second;
+            }) != vals.end()) {
```

Comment:
I think we should remove this check: Bin sizes can be changed *after* creation without checks, so we should throw here even if there are not empty buckets?

Reply:
That would be very conservative. But then we would be on the safe side.

Reply:
Actually, the error message doesn't say anything about empty bins...